question-1.md

```diff
@@ -5,7 +5,11 @@ number: 1
 type: radio
 ---
 
-Examples:
+**Examples of covered entities:**
 
-- example 1
-- example 2
\ No newline at end of file
+- Health care providers <br>
+(Doctors, clinics, psychologists, dentists, chiropractors, nursing homes, pharmacies)
+-  Health plans <br>
+(Health insurance companies, HMOs, company health plans, government health programs such as Medicare, Medicaid, and the military and veterans health care programs)
+- Health care clearinghouse <br>
```

Comment:
I'm still not sure what this is. We may need to provide more information or completely reword this to avoid the word "clearinghouse." Probably makes people think of "sweepstakes," which is probably incorrect and not helpful. :)